### ISSUE TITLE
[Bug] (MacOS) Dock Badge for Unread Messages doesn't work.

### ISSUE THREAD

**Issue by zxzzz8** [oxen-io/session-desktop#2590]
using the mac version of session, there is no indicator in the dock for unread messages....
i've enabled notifications and everything, both in the app and in the system, but it still doesn't work.

**Comment by KeeJef** [comment]
@yougotwill  maybe something to look into

**Comment by Bilb** [comment]
yeah, I guess this is about the Badge Count, which we are presently not setting
https://www.electronjs.org/docs/latest/api/app#appsetbadgecountcount-linux-macos

We'd need to add logic for that on the IPC layer and debounce the calls so we do send one update for every new message
